src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenExprLambdaTests.cs

```diff
@@ -10,6 +11,26 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
 {
     public class CodeGenExprLambdaTests : CSharpTestBase
     {
+        //TODO: 4.6 should be the default
```

Comment:
yes. I think 4.6 should be the default for most tests, but do not want to do that in this change.